pallets/api/src/fungibles/tests.rs

```diff
@@ -297,43 +483,56 @@ fn signed(account: AccountId) -> RuntimeOrigin {
 	RuntimeOrigin::signed(account)
 }
 
-fn create_token(owner: AccountId, token: TokenId) {
+fn root() -> RuntimeOrigin {
+	RuntimeOrigin::root()
+}
+
+fn none() -> RuntimeOrigin {
+	RuntimeOrigin::none()
+}
+
+fn pallet_assets_create(owner: AccountId, token: TokenId) {
```

Comment:
Mentioned it somewhere that it is better to have all these methods organized under a module `pallet_assets`. Guess you already tackled that in the another PR.